paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/sink/MultiTablesStoreCompactOperator.java

```diff
@@ -165,7 +168,8 @@ public void processElement(StreamRecord<RowData> element) throws Exception {
                     files.isEmpty(),
                     "Batch compact job does not concern what files are compacted. "
                             + "They only need to know what buckets are compacted.");
-            write.compact(partition, bucket, true);
+            // `minor` compact strategy is supported in batch mode.
```

Comment:
`minor` and `full` compact type are both supported in batch mode.

Reply:
Yeah, this reminder should be placed above.  `full compact is not supported in streaming mode.`